Languages/en_US/ManageMaintenance.php

```diff
@@ -42,7 +42,7 @@
 $txt['repair_missing_topic_for_cache'] = 'Cached word "{0}" is linked to a non-existent topic.';
 $txt['repair_missing_log_poll_member'] = 'Poll #{0, number, integer} has been given a vote from member #{1, number, integer} , who is now missing.';
 $txt['repair_missing_log_poll_vote'] = 'A vote was cast by member #{0, number, integer} on a non-existent poll #{1, number, integer}.';
-$txt['repair_missing_thumbnail_parent'] = 'A thumbnail exists called {0}, but it does not have a parent.';
+$txt['repair_missing_thumbnail_parent'] = 'A thumbnail exists called {0}, but it doesn\'t have a parent.';
```

Comment:
```suggestion
$txt['repair_missing_thumbnail_parent'] = 'A thumbnail exists called {0}, but it does not have a parent.';
```

Unfortunately, a single apostrophe is a [syntax character in ICU MessageFormat](https://unicode-org.github.io/icu/userguide/format_parse/messages/#quotingescaping).

Therefore, we should avoid using apostrophes where we can. In situations where we must use one, we should use the real Unicode apostrophe character `’` (U+2019).